lab1.js

```diff
@@ -99,12 +96,22 @@ assert(sentence2 === 'chirp chirp chirp chirp chirp chirp chirp chirp chirp.',
 */
 
 var favoriteAnimals = ['elephant', 'penguin', 'eagle', 'camel'];
-var nextAnimal;
 
-// TODO: 12 points
-// Assign one of your favorite animals to nextAnimal using Math.random() to pick
+var nextAnimal = Math.random();
 
-// your code goes here
+if (nextAnimal < 0.25) {
+  nextAnimal = favoriteAnimals[0];
```

Comment:
Is the preferred method in industry to make things whole numbers? The mathematician in me love decimals.